mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
Good point.

How likely is it that System.Net.Sockets code and Mono.Unix.Native code will be intermixed? What's the use case there? Meaning, is this a likely naming collision? (It might not be.)

I just find "Sock" to be an ugly prefix.

Alternatively, we could do `NativeSocketType`, `NativeSocketFlags`, etc., but that's similarly fugly.